app/containers/message/index.js

```diff
@@ -171,38 +206,46 @@ export default class MessageContainer extends React.Component {
 		} = item;
 		const isEditing = editingMessage._id === item._id;
 		return (
-			<Message
-				msg={msg}
-				author={u}
-				ts={ts}
-				type={t}
-				status={status}
-				attachments={attachments}
-				urls={urls}
-				reactions={reactions}
-				alias={alias}
-				editing={isEditing}
-				header={this.isHeader()}
-				avatar={avatar}
-				user={user}
-				edited={editedBy && !!editedBy.username}
-				timeFormat={this.timeFormat}
-				style={style}
-				archived={archived}
-				broadcast={broadcast}
-				baseUrl={baseUrl}
-				customEmojis={customEmojis}
-				reactionsModal={reactionsModal}
-				useRealName={useRealName}
-				role={role}
-				closeReactions={this.closeReactions}
-				onErrorPress={this.onErrorPress}
-				onLongPress={this.onLongPress}
-				onReactionLongPress={this.onReactionLongPress}
-				onReactionPress={this.onReactionPress}
-				replyBroadcast={this.replyBroadcast}
-				toggleReactionPicker={this.toggleReactionPicker}
-			/>
+			<Swipeable
```

Comment:
This entire visual logic should be moved to `Message.js` and it should be a default action (without needing swipe a touch to perform a reply).

Reply:
I could not understand what you mean by default action, I just copied exactly like WhatsApp you swipe you trigger the action

Reply:
@pranavpandey1998official seeing your gif, there's a delay between swipe and reply. I assume this is a touch on the reply. It's not, then? A swipe opens the reply?

Reply:
yes swipe opens reply, yes there is a lot delay may be because of whole action dispatch thing.

Reply:
done